docs/devcontainer-spec-support.md

```diff
@@ -0,0 +1,72 @@
+# Support for Dev Container Specification
+
+> Refer to the full Dev Container specification [here](https://containers.dev/implementors/json_reference/) for more information on the below options.
+
+The symbols in the first column indicate the support status:
+
+- 🟢 Fully supported.
+- 🟠 Partially supported.
+- 🔴 Not currently supported.
+
+The last column indicates any currently existing GitHub issue for tracking support for this feature.
+Feel free to [create a new issue](https://github.com/coder/envbuilder/issues/new) if you'd like Envbuilder to support a particular feature.
+
+| Status                        | Name                                  | Description                                                                                                            | Known Issues                                           |
+| ----------------------------- | ------------------------------------- | ---------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------ |
+| **General**                   |                                       |                                                                                                                        |
```

Comment:
should we present supported features first?

Reply:
I'm going to punt on ordering for now as it's fairly easy to change in a separate PR.